tests/pytest/test.py

```diff
@@ -381,44 +381,38 @@ def testMgetCommand(env):
     """Test REJSON.MGET command"""
     r = env
 
-    # Skip on Redis Unstable
-    _version = "99"
-    res = r.con.execute_command('INFO')
-    if(version.parse(res['redis_version']) >= version.parse(_version)):
```

Comment:
If we don't need to check the version, then I think we can remove the import:
``from packaging import version``